java/src/main/native/src/csv_chunked_writer.hpp

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include "jni_writer_data_sink.hpp"
+
+#include <cudf/io/csv.hpp>
+
+#include <cassert>
+
+namespace cudf::jni::io {
```

Comment:
Please be consistent with the namespace. If we are going to use `...::io` then it should be added anywhere relevant.

Reply:
I would like for the IO classes to be moved to the `cudf::jni::io` namespace. There might be value in moving `jni_writer_data_sink` here as well.

Reply:
I'll add this to the followup issue #12456 I'm working on.

Reply:
#12458 addresses the use of `cudf::jni::io`.